email.go

```diff
@@ -1,10 +1,21 @@
 package packngo
 
+import "fmt"
+
 const emailBasePath = "/emails"
 
+// EmailCreateRequest type used to add an email address to the current user
+type EmailRequest struct {
```

Comment:
I think you will have to make the fields in this struct pointers. Without that, it's impossible to make default email nondefault. Imagine you want to update an email like
```
c.Emails.Update(id, &EmailUpdate{Default: false})
```

Golang will serialize this as `{}` to the HTTP request because `false` is a default value for `bool`, and the field is tagged as `omitempty`. At the same time, you can't drop the omitempty, because then `"default":false` would be present in every Update request, for example only changing the email string.

If you make it a pointer and keep it `omitempty`, the field will be populated only if the value is non-nil (nil is default for pointers). I think this is the desired behavior.

So this goes for any struct that corresponds to API resource that can be updated - it's because of the way Golang deals with JSON.